app/config/errors.php

```diff
@@ -364,6 +364,28 @@
         'code' => 416,
     ],
 
+    /** VCS */
+    Exception::INSTALLATION_NOT_FOUND => [
+        'name' => Exception::INSTALLATION_NOT_FOUND,
+        'description' => 'Installation with the requested ID could not be found.',
+        'code' => 404,
+    ],
+    Exception::PROVIDER_REPOSITORY_NOT_FOUND => [
+        'name' => Exception::PROVIDER_REPOSITORY_NOT_FOUND,
+        'description' => 'Repository with the requested ID could not be found.',
+        'code' => 404,
+    ],
+    Exception::REPOSITORY_NOT_FOUND => [
+        'name' => Exception::REPOSITORY_NOT_FOUND,
+        'description' => 'VCS Repository with the requested ID could not be found.',
+        'code' => 404,
+    ],
```

Comment:
whats the difference between these two errors ?

Reply:
We prefix `Provider` everything that is related to VCS provider. So First one would be when GitHub repository cannot be found, and the second one if when appwrite repository document cannot be found.

Update descriptions to better terms regarding "vcs" and "provider"